tldw_Server_API/app/main.py

```
@@ -359,6 +359,19 @@ def _startup_trace(msg: str) -> None:
             logger.debug(f"Startup trace logging failed: {_startup_log_err}")
 
 _startup_trace(f"Endpoint import gating: ULTRA_MINIMAL_APP={_ULTRA_MINIMAL_APP}, MINIMAL_TEST_APP={_MINIMAL_TEST_APP}")
+
+# Detect whether we're running a pytest test that lives under tests/Jobs/
+def _in_jobs_test_context() -> bool:
+    try:
+        node = os.getenv("PYTEST_CURRENT_TEST", "")
+        if not node:
+            return False
+        node = node.replace("\\", "/").lower()
+        return "/tests/jobs/" in node
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)